### PR TITLE
Some minor error fixes.

### PR DESCRIPTION
**Pull request by david-deboer**
This fixed the handful of small errors I found in looking through the code.  I had also done a few tweaks in the new methods.

### CODE REVIEW

**Comment by david-deboer** [comment]
My editor must also automatically remove the trailing spaces...

File: control_software/src/blocks.py [hunk 1]
```diff
@@ -64,17 +64,12 @@ def _critical(self, msg, *args, **kwargs):
...
-        return a list of all register names within
-        the block.
-        """
-
     def initialize(self, verify=False):
         """
         Individual blocks should override this
         method to configure themselves appropriately
         """
+        print("THIS DOESN'T DO ANYTHING")
```
Comment (AaronParsons) [inline control_software/src/blocks.py:72]: Put a "pass" here instead of printing.

File: control_software/src/hera_corr.py [hunk 2]
```diff
@@ -815,7 +815,7 @@ def eq_initialize(self, host, verify=True):
                     failed.append((stream, (ant, pol)))
         if len(failed) > 0:
             raise RuntimeError('Failed to initialize EQ on %s: %s' %
-                               (host, ','.join(['%d=(%d%s)' % (ant,pol)
+                               (host, ','.join(['(%d%s)' % (ant,pol)
```
Comment (AaronParsons) [inline control_software/src/hera_corr.py:818]: The first %d looks like it should have been *stream*. Can you put it back and set it to stream?

**Review by AaronParsons — CHANGES_REQUESTED** [review]
Two minor suggestions to a couple of edits. Otherwise looks great. good catches.

**Review by AaronParsons — APPROVED** [review]
All in order. Next time, can we separate code cleanup from new functionality?

**Pull request merged**